lms/djangoapps/grades/models.py

```diff
@@ -380,6 +381,7 @@ class Meta(object):
         # (user_id) for course dashboard; explicitly declared as an index below
         unique_together = [
             ('course_id', 'user_id'),
+            ('date_passed', 'course_id'),
```

Comment:
Don't we need to include the user here? It doesn't seem unreasonable for multiple users in a given course to have the same `data_passed` value.